browser/sign.js

```diff
@@ -19,6 +21,7 @@ function sign (hash, key, hashType, signType, tag) {
   } else {
     if (signType !== 'rsa' && signType !== 'ecdsa/rsa') throw new Error('wrong private key type')
   }
+  if (key.padding !== undefined && key.padding !== RSA_PKCS1_PADDING) throw new Error('padding scheme not supported')
```

Comment:
can you add some tests for this? What does node do in this case?

Reply:
Node.js also [supports](https://nodejs.org/api/crypto.html#signsignprivatekey-outputencoding) `RSA_PKCS1_PSS_PADDING`. If the padding value is incorrect, the [call](https://github.com/nodejs/node/blob/18e00a577d7405ca0e6c26113675c230677342c2/src/crypto/crypto_sig.cc#L64) of OpenSSL's [`EVP_PKEY_CTX_set_rsa_padding`](https://www.openssl.org/docs/manmaster/man3/EVP_PKEY_CTX_set_rsa_pss_saltlen.html) returns a negative value and Node.js presumably throws an exception.

Reply:
Great - then i should be able to verify that with the test case for this package :-)